src/interface/report/PlayerLoader.js

```diff
@@ -211,16 +211,16 @@ class PlayerLoader extends React.PureComponent {
   }
 
   renderLoading() {
-    return <ActivityIndicator text={i18n._(t`Fetching player info...`)} />;
+    return <ActivityIndicator text={i18n._(t('interface.report.renderLoading.FetchingPlayerInfo')`Fetching player info...`)} />;
```

Comment:
```suggestion
    return <ActivityIndicator text={i18n._(t('interface.report.renderLoading.fetchingPlayerInfo')`Fetching player info...`)} />;
```
very minor but I think everything should be in camelCase